core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogItemDtoAbstract.java

```diff
@@ -45,6 +45,8 @@
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Sets;
 
+// TODO add support for nested catalog items, implement nestCatalogItemId in terms of symbolicName/Version
+// TODO also getCatalogItemSuperIds.
```

Comment:
I think we should either override with NOPs, or just keep the default behaviour from `AbstractBrooklynObject`. The catalog items don't know what's the meaning of the plan in them.